### ISSUE TITLE
Handling double quote issue

### ISSUE THREAD

**Issue by Jaryn91** [JoshClose/CsvHelper#1916]
I got CSV file that I need to parse. I don't have any influence on my csv input.
CSV is comma-separated.

`List<MagazineCsv> list = new List<MagazineCsv>();
var config = new CsvConfiguration(CultureInfo.InvariantCulture)
{
    Delimiter = ","
};
using (var reader = new StreamReader(csvPath))
using (var csv = new CsvReader(reader, config))
{
    var bad = new List<string>();
    var isRecordBad = false;

    while (csv.Read())
    {
        var record = csv.GetRecord<MagazineCsv>();
        list.Add(record);
    }
}`

Generally, it works fine. There is a problem when I got CSV with a double-quote like that:

`12-11-2021,10-11-2021,Bought  "SuperCar" - No Return,,,"-775,00",,25,`

When I run a program I got an error: **You can ignore bad data by setting BadDataFound to null.**

I searched for a solution for that issue to make it parse like **Bought  "SuperCar" - No Return**, but without success. Few people have already gone thru that problem as well. I found out that is not the correct format of CSV. It would be nice to add an option in the library to overcome that double-quote problem.
I really like your csvHelper library and I want to keep using that but, I don't know how to nicely overcome that problem.

**Comment by i2van** [comment]
```csharp
var config = new CsvConfiguration(CultureInfo.InvariantCulture)
{
  Delimiter = ",";
  BadDataFound = null; // Try this. WARNING: it might mangle CSV parsing.
};
```

**Comment by proca-ainq** [comment]
I don't think that just ignoring bad data is the right approach here because you may actually need to handle other things in the BadDataFound event.  +1 to the OP because this double-quote problem has been problematic for us as well.  The library can't handle un-escaped double-quotes but when you open it in LibreOffice, it works fine.  There should be better handling of un-escaped double-quotes if the rest of the row can be validated with the expected columns.

**Comment by MithrilMan** [comment]
why not add an option to explicitly handle double quotes if this is a common enough scenario?

**Comment by ingbond** [comment]
Same issue.
Mode is CsvMode.RFC4180
All is fine before parser meet such row:
...; From weep hole "breather ";...
It can't handle double quotes
